examples/julia_set/Cargo.toml

```diff
@@ -8,4 +8,3 @@ crate-type = ["cdylib"]
 
 [dependencies]
 wasm-bindgen = { path = "../.." }
-web-sys = { path = "../../crates/web-sys" }
```

Comment:
Wait -- why?

Reply:
Ah we talked about this on IRC but I was overzealous at the start of this PR and this ended up not working out, so this is me backing out intermediate changes